opus/import/obs_bundle_cassini_uvis_solarocc_beckerjarmak2023.py

```
@@ -0,0 +1,243 @@
+
+    def field_obs_ring_geometry_projected_radial_resolution1(self):
+        return None
+
+    def field_obs_ring_geometry_projected_radial_resolution2(self):
+        return None
+
+    # The solar ring elevation, observer ring elevation, phase, incidence angle, and
+    # emission angles for Becker/Jarmak are calculated in obs_bundle_occ_common.py.
+    # However, the north-based fields are specific to planet and geometry:
+
+    def _north_based_angle_helper(self):
```

Ruff (0.14.5)

Missing return type annotation for private function `_north_based_angle_helper`

(ANN202)